packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -54,7 +54,7 @@ export default class ContentScript {
       )
     }
 
-    const exposedMethods = {}
+    const exposedMethods = null
```

Comment:
In the following `for` we have `exposedMethods[method] = ...` so I suppose we'll have an null exception?

Reply:
You should be able to do this
```js
/** @type {import('post-me').MethodsType} */
const exposedMethods = {}
```